app/models/user.rb

```diff
@@ -0,0 +1,11 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  has_many :categories
+  has_many :transactions
+
+  validates :name, presence: true
+  validates :email, presence: true
```

Comment:
- [OPTIONAL] Great job using devise for authentication however you need to validate all user input  (password, password confirmation) to make sure that anyone with bad intentions cannot compromise their app. Please add this 👍🏻